### PR TITLE
i18n(es): update `components`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description

- What does this PR change? Give us a brief description.
Updates `components` based on these commits:
  - 68f56a7ffd314b760443a057db9ed1a982dbe191
  - caa84eaa7dc653d27d539fd3a93df346a9f0f149

### CODE REVIEW

File: docs/src/content/docs/es/guides/components.mdx [hunk 1]
```diff
@@ -227,6 +227,38 @@ import { LinkCard } from '@astrojs/starlight/components';
...
+<LinkButton href="https://docs.astro.build" variant="secondary" icon="external">
+	Relacionado: Astro
+</LinkButton>
+```
+
+El código anterior genera lo siguiente en la página:
+
+import { LinkButton } from '@astrojs/starlight/components';
+
+<LinkButton href="/getting-started/">Comienza</LinkButton>
```
Comment (delucis) [inline docs/src/content/docs/es/guides/components.mdx:257]: This code example tricked you 😁 Needs an `/es/` link here too.

````suggestion
<LinkButton href="/es/getting-started/">Comienza</LinkButton>
<LinkButton href="https://docs.astro.build" variant="secondary" icon="external">
	Relacionado: Astro
</LinkButton>
```

El código anterior genera lo siguiente en la página:

import { LinkButton } from '@astrojs/starlight/components';

<LinkButton href="/es/getting-started/">Comienza</LinkButton>
````

**Review by delucis — COMMENTED** [review]
Thanks @dreyfus92! Looks like there’s one link issue.

**Comment by dreyfus92** [comment]
Thanks for the review Chris 😁

**Review by delucis — APPROVED** [review]
Thank you for all the PRs! 💖

**Pull request merged**